release-engineering/role-handbooks/branch-manager.md

```diff
@@ -292,77 +310,146 @@ The following are some ways to determine if the release process was successful:
+    ```
 
-1. `grep` for [`manual-release-bump-required`](https://github.com/kubernetes/test-infra/search?q=manual-release-bump-required&unscoped_q=manual-release-bump-required)` in [test-infra](https://github.com/kubernetes/test-infra) and duplicate block entries appropriately, i.e. in:
-   1. [`config/jobs/kubernetes/sig-gcp/gce-conformance.yaml`](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-gcp/gce-conformance.yaml)
-   1. [`testgrid/config.yaml`](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml)
+13. Issue a PR with the new release branch job configurations ([example PR](https://github.com/kubernetes/test-infra/pull/13882))
 
-1. Verify that Testgrid is in a reasonable state. As of today, this means deleting the oldest `sig-release-<version>-{master,informing,all}` dashboards e.g. `sig-release-1.12-all` and making new `sig-release-<version>-{master,informing,all}` ones in [`testgrid/config.yaml`](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml). You can run `bazel test //testgrid/cmd/configurator/...` to make sure you are in a good state.
+14. Once the PR has merged, verify that the new dashboards have been created and are populated with jobs:
+    - https://testgrid.k8s.io/sig-release-1.16-all
+    - https://testgrid.k8s.io/sig-release-1.16-blocking
+    - https://testgrid.k8s.io/sig-release-1.16-informing
 
-1.  Finally, update the [release target section](https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs) of the README.
+15. [Announce in #sig-release and #release-management](https://kubernetes.slack.com/archives/C2C40FMNF/p1565746110248300?thread_ts=1565701466.241200&cid=C2C40FMNF) that this work has been completed
 
 [sig-release-1.Y-blocking]: https://testgrid.k8s.io/sig-release-1.15-blocking
 [`branchff`]: https://git.k8s.io/release/branchff
 
-# Configure Merge Automation
+## Configure Merge Automation
 
 Starting release cycle v1.16, branch management will take on duties from the Test Infra lead. Since the role was dissolved at the end of cycle v1.15. This section describes the added responsibility that branch management will partake:
 
-Between the Code Freeze and lifting Code Freeze (or Code Thaw) period, merging new code is restricted (focus is on fixing existing code and getting green test builds on Testgrid) and is implemented by a combination of prow plugins config, submit-queue config, and [tide]. The `master` and current release cycle branch i.e. release-X.Y are the only branches affected during this period.
+Between the Code Freeze and lifting Code Freeze (or Code Thaw) period, merging new code is restricted (focus is on fixing existing code and getting green test builds on Testgrid) and is implemented by a combination of prow plugins config, submit-queue config, and [tide]. The `master` and current release cycle branch i.e. release-x.y are the only branches affected during this period.
```

Comment:
I don't think we change any plugin configs to enforce anything since 1.14 onward when we dropped code slush as a concept, and we *_super definitely_* don't use the submit queue 👻 anymore
```suggestion
Between the Code Freeze and lifting Code Freeze (or Code Thaw) period, merging new code is restricted (focus is on fixing existing code and getting green test builds on Testgrid) and is implemented by config changes for [tide]. The `master` and current release cycle branch i.e. release-x.y are the only branches affected during this period.
```